### ISSUE TITLE
Experiment with multi-tab / multi-window models.

### ISSUE THREAD

**Issue by tshead2** [sandialabs/slycat#418]

**Comment by alexsielicki** [comment]
@tshead2 have you explored using localStorage for the communication between tabs/windows? 

http://stackoverflow.com/questions/2236828/javascript-communication-between-tabs-windows-with-same-origin/12514384#12514384

https://github.com/diy/intercom.js/

**Comment by tshead2** [comment]
I haven't; I think it might simplify some issues (eliminate the need to get open window handles, for example), while leaving others (you still have to get the windows open, if you're opening a bookmark with multiple windows).

**Comment by alexsielicki** [comment]
This layout library might be useful when implementing this, or if we need to replace the current layout system:
http://golden-layout.com

**Comment by Mletter1** [comment]
Yeah I’m currently using golden layout on anouther project and like using it
